src/main/kotlin/org/taktik/icure/services/external/rest/v2/dto/embed/TimeTableItemDto.kt

```diff
@@ -26,11 +26,12 @@ import com.github.pozo.KotlinBuilder
 @JsonIgnoreProperties(ignoreUnknown = true)
 @KotlinBuilder
 data class TimeTableItemDto(
-	val days: List<String> = emptyList(),
+	val rrule: String? =null,
+	@Deprecated("Will be replaced by rrule") val days: List<String> = emptyList(),
```

Comment:
In V2, instead of deprecating the fields, as they already are deprecated in V1, delete them from the V2 DTOs

Reply:
The code won't  compile if I delete this.

Reply:
What do you mean by it won't compile ? If you delete it and adapt the TimeTableController it will. 
The goal of the v2 API is to add new functionalities but also to clean the v1 API. It doesn't make any sense to keep deprecated fields to keep it compiling, it is the duty of the v1. 
Continue using the v1 API to support the legacy system, but when you go to v2, update your system to the new logic. 
Therefore, I will ask you to delete the deprecated fields from the v2 DTOs.